src/librustc_const_eval/eval.rs

```diff
@@ -68,7 +68,7 @@ pub fn lookup_const_by_id<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>,
             _ => Some((def_id, substs))
         }
     } else {
-        match tcx.sess.cstore.describe_def(def_id) {
+        match ty::queries::describe_def::get(tcx, DUMMY_SP, def_id) {
```

Comment:
This might be a dumb question but, when implementing a provider in `src/librustc/ty/mod.rs`, why don't we just call the query in this manner and passing it `tcx`?  

It seems to provide the same type and perform a similar function.